src/js/jsx/DocumentHeaderTab.jsx

```diff
@@ -59,7 +59,7 @@ define(function (require, exports, module) {
                     })}
                     title={this.props.name}
                     onClick={this.props.onClick}>
-                    {this.props.dirty ? "•" : ""}
+                    {this.props.dirty ? "*" : ""}
```

Comment:
Seriously though, I'm not saying the bullet was good, but is an asterisk really the desired symbol here!?